core/src/test/scala/org/apache/spark/launcher/LauncherBackendSuite.scala

```diff
@@ -17,15 +17,16 @@
 
 package org.apache.spark.launcher
 
+
 import java.util.concurrent.TimeUnit
 
 import scala.concurrent.duration._
 import scala.language.postfixOps
 
-import org.scalatest.Matchers
 import org.scalatest.concurrent.Eventually._
+import org.scalatest.Matchers
```

Comment:
Please avoid making unnecessary changes to imports. This one, in particular, is wrong and would trigger a style checker error if the style checker didn't have a bug...